ngraph/test/runtime/ie/ie_executable.cpp

```diff
@@ -93,9 +93,6 @@ runtime::ie::IE_Executable::IE_Executable(shared_ptr<Function> func, string devi
     : m_device{device}
 {
     static std::set<NodeTypeInfo> ie_ops = get_ie_ops();
-    pass::Manager passes;
```

Comment:
Need to double-check if IE_Backend is still used somewhere. If so, we haven't to change its behavior.  Also, I've reverted opset1_upgrade.hpp in the update_evaluates target branch